tests/integration/features/backup_ondemand_psmdb/31-assert.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: kuttl.dev/v1
+kind: TestAssert
+timeout: 100
+collectors:
+  - command: kubectl get db/test-psmdb-cluster -n ${NAMESPACE} -o yaml
+  - command: kubectl get psmdb/test-psmdb-cluster -n ${NAMESPACE} -o yaml
+
+  - command: kubectl get deploy/everest-controller-manager -n everest-system -o yaml
+  - type: pod
+    namespace: everest-system
+    selector: control-plane=controller-manager
+    tail: 100
+resourceRefs:
+  - apiVersion: everest.percona.com/v1alpha1
+    kind: DatabaseCluster
+    name: test-psmdb-cluster
+    ref: db
+  - apiVersion: psmdb.percona.com/v1
+    kind: PerconaServerMongoDB
+    name: test-psmdb-cluster
+    ref: psmdb
+---
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseCluster
+metadata:
+  name: test-psmdb-cluster
+spec:
+  engine:
+    crVersion: 1.20.1
```

Comment:
better to use ENV var for this like here https://github.com/percona/everest-operator/blob/c147d08fe726f800107ec3a8e7c770ed4a9532d2/tests/integration/core/psmdb/10-assert.yaml#L14
```
commands:
 - command: kubectl wait --for=jsonpath='{.spec.engine.crVersion}'=${PSMDB_OPERATOR_VERSION} dbc/test-psmdb-cluster -n ${NAMESPACE}
```